### ISSUE TITLE
Empty select option renders too small

### ISSUE THREAD

**Issue by Gummibeer** [laravel-filament/filament#282]
**Describe the bug**
If a select option has no value/label it has a much less height than others. In select dropdown but also if selected.

Additionally I would also expect `nullable()` on a select to add an empty option. 🤔

**To reproduce**

```php
Components\Select::make('my_select')
    ->options([null => '', 'foo' => 'bar'])
    ->nullable(),
```

**Expected behavior**
Same height select option as with content. Personally I think that a native `<select>` would be totally enough by default!? 🤔

**Screenshots**
open
<img width="1096" alt="Bildschirmfoto 2021-03-23 um 19 38 16" src="https://user-images.githubusercontent.com/6187884/112205447-13d72200-8c15-11eb-8b9a-4cf9bd2f111d.png">
hover
<img width="1096" alt="Bildschirmfoto 2021-03-23 um 19 38 22" src="https://user-images.githubusercontent.com/6187884/112205444-13d72200-8c15-11eb-9d4e-7a4f97640dcb.png">
selected
<img width="1092" alt="Bildschirmfoto 2021-03-23 um 19 38 29" src="https://user-images.githubusercontent.com/6187884/112205439-12a5f500-8c15-11eb-97ec-8a8548ea2cd4.png">

**Comment by yob-yob** [comment]
isn't it a good UX if you put a placeholder for the null value?

If I'm a user, I wouldn't know what that blank option is.. is it a bug? maybe something should show there but it isn't? is it just empty? I don't know...

can you try using `->placeholder($placeholder);` ?

**Comment by Gummibeer** [comment]
It's an admin panel and not a user-facing view. So an admin complaining about "I don't know what the empty option means" will be fired. 😉
I expect a custom select to behave 100% equal or better than a default HTML one. And a vanilla HTML one doesn't reduce it's own height.

**Comment by danharrin** [comment]
It's not the best idea to use it without a placeholder, but nonetheless this is a UI bug.

**Comment by danharrin** [comment]
Closed by #340.
